symbolic-debuginfo/Cargo.toml

```
@@ -42,8 +42,8 @@ serde_json = "1.0.40"
 smallvec = "1.2.0"
 symbolic-common = { version = "8.1.0", path = "../symbolic-common" }
 thiserror = "1.0.20"
-walrus = "0.18.0"
-wasmparser = "0.59.0"
+walrus = "0.19.0"
+wasmparser = "0.78.2"
```

Please align the dependencies with eachother. `walrus 0.19` requires `wasmparser ^0.77.0`

```suggestion
wasmparser = "0.77.0"  # must match spec in walrus
```